deepchecks/nlp/checks/model_evaluation/prediction_drift.py

```diff
@@ -150,10 +152,16 @@ def run_logic(self, context: Context) -> CheckResult:
         # Flag for computing drift on the probabilities rather than the predicted labels
         proba_drift = ((len(context.model_classes) == 2) and (self.drift_mode == 'auto')) or \
                       (self.drift_mode == 'proba')
+        model_classes = context.model_classes
 
         if proba_drift:
+            if context.is_multi_label_task():
+                raise DeepchecksValueError('Cannot use proba drift mode for multi-label tasks')
```

Comment:
why? You have multilabel proba, can't flatten?

Reply:
we dont have multi label proba...

Reply:
Looking at the context + dummy model seems like we do. Why you say we don't? Tried it and it doesn't work for some reason?

Reply:
yea you right, saw it in yuri PR (ps. i dont think it will work with None) 
anyhow proba drift is not important enough to spend time on now

Reply:
Agree in principle, but why would simple flattening won't work?